profiles/models.py

```diff
@@ -297,9 +311,19 @@ class VerifiedPersonalInformationPermanentForeignAddress(
 ):
     RELATED_NAME = "permanent_foreign_address"
 
-    street_address = fields.EncryptedCharField(max_length=1024, blank=True)
-    additional_address = fields.EncryptedCharField(max_length=1024, blank=True)
-    country_code = fields.EncryptedCharField(max_length=3, blank=True)
+    street_address = fields.EncryptedCharField(
+        max_length=1024,
+        blank=True,
+        help_text="Street address or whatever is the _first part_ of the address.",
+    )
+    additional_address = fields.EncryptedCharField(
+        max_length=1024,
+        blank=True,
+        help_text="Additional address information, perhaps town, county, state, country etc.",
+    )
+    country_code = fields.EncryptedCharField(
+        max_length=3, blank=True, help_text="An ISO 3166-1 country code."
```

Comment:
Could specify if this is supposed to be two or three character country code.

Reply:
Yes. Originally I made these fields very lax. That's where the documentation comes from. There's also a plan (HP-436) to make these fields more specified and stricter. The documentation needs to be revisited at that point for every field.

Reply:
With this country code field the situation is a bit more involved. The plan is that it accepts (via the GraphQL API at least) all versions of country codes, alpha-2, alpha-3 and numeric-3. But then a translation is made to the alpha-2 format and that's also the output format. But we are not there yet.